load/LPTSExtractReader.py

```diff
@@ -812,6 +848,10 @@ def Has_Ambiguous_Agreement(self):
 		total = self.LORA() + self.Text_Agreement() + self.Other_Agreement()
 		return total > 1
 
+	def Has_Ambiguous_Methodology(self):
+		total = self.TraditionalRecording() + self.VirtualRecording() + self.Partner() + self.Joint() + self.HearThis() + self.Render()
+		return total > 1
```

Comment:
I think this should be changed to 
return total != 1

Because it is also ambiguous if there is no methodology

Reply:
@bradflood , I have updated it as you can see in the following url: https://github.com/faithcomesbyhearing/dbp-etl/pull/135/files#diff-943fbc0f60984b469ccd792a22ead44a9b9eaa1c4e9050b1433f5997c812d252R853